.github/workflows/pre-commit.yml

```diff
@@ -19,16 +19,16 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: 3.11.0
+        python-version: 3.10.8
```

Comment:
Why exactly 3.10.8 instead of the more recent 3.11?

Reply:
3.10.8 is the version that is the most recent version that we get on balfrin in the user environment, which is used also by buildbot. So we are testing the same version that we use on balfrin. Once we made probtest independent from that we can use more recent version.

Reply:
I see, ok makes sense :+1: